InspectCode.ps1

```diff
@@ -1,27 +0,0 @@
-[CmdletBinding()]
```

Comment:
Not sure about just removing `InspectCode.ps1` altogether, since it's something that people that don't have a jetbrains IDE/resharper tend to be told to use rather than hammering CI, to the point where it's even name-dropped in README. Should be easy enough to keep the script but replace its contents with dotnet tool calls I think?

Reply:
That's fair. I've added the script back + a bash version.